src/trace_processor/perfetto_sql/stdlib/linux/perf/etm.sql

```diff
@@ -0,0 +1,41 @@
+--
+-- Copyright 2025 The Android Open Source Project
+--
+-- Licensed under the Apache License, Version 2.0 (the 'License');
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     https://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an 'AS IS' BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+-- Given a trace_id, returns the file_name and relative program counter for
+-- each instruction in that trace.
+CREATE PERFETTO FUNCTION __linux_perf_etm_metadata(
```

Comment:
THis should also be a single underscore. Double underscore are for c++ implementations.